api/v1alpha1/ngrok_common.go

```diff
@@ -1,7 +1,7 @@
 package v1alpha1
 
 // common ngrok API/Dashboard fields
-type ngrokAPICommon struct {
+type NgrokAPICommon struct {
```

Comment:
for some reason when I was coding it thought i had issues creating the struct literal and setting these fields. I must have mixed something up though because i moved this back to private and things work

Reply:
ahh, it was when i was trying to set it when creating the struct. But since i changed to set it explicitly after being created, its happy
![image](https://user-images.githubusercontent.com/8029578/221919950-467ba17f-99c4-4363-a2c5-bfc8704852f6.png)